script/DeploymentScript.s.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import {ModuleBeaconDeployer_v1} from "script/ModuleBeaconDeployer_v1.s.sol";
+
+import {Governor_v1} from "@ex/governance/Governor_v1.sol";
+import {FeeManager_v1} from "@ex/fees/FeeManager_v1.sol";
+import {ModuleFactory_v1} from "src/factories/ModuleFactory_v1.sol";
+import {OrchestratorFactory_v1} from "src/factories/OrchestratorFactory_v1.sol";
+
+contract DeploymentScript is ModuleBeaconDeployer_v1 {
+    //@note not upgradable right?
+    address public inverterReverter = ext_InverterReverter_v1;
+    address public governor;
+    address public forwarder;
+    address public feeManager;
+
+    address public moduleFactory;
+    address public orchestratorFactory;
+
+    function run() public {
+        console2.log(
+            "--------------------------------------------------------------------------------"
+        );
+        console2.log("Start Deployment");
+
+        //Create External Singletons
```

Comment:
General thing: Please leave a space between `//` and the comment :D If you want to have an easy way to not change your habits and still provide code that doesn't trigger me (😂) here is a regex command that can help you detect any of these without false positives.

```regex
(?<!:)\/\/(?!\s|\/|-{2,})
```

Just put it into the search on VSCode, click the "regex" button that looks like `.*` and exclude `lib`. Then just put `// ` (with a whitespace after the //) into the replace line and go for it. 

![Screenshot 2024-08-01 at 13 12 07](https://github.com/user-attachments/assets/bfe67ff3-b500-4a08-9d59-975998d59821)

Reply:
And sorry in advance for this 😂

Reply:
lol

Reply:
![dog-trigger](https://github.com/user-attachments/assets/1c19629a-5ea5-4f35-b96d-378a83366c76)